libs/cohere/langchain_cohere/llms.py

```diff
@@ -92,18 +92,22 @@ def validate_environment(self) -> Self:  # type: ignore[valid-type]
             cohere_api_key = self.cohere_api_key
         client_name = self.user_agent
         timeout_seconds = self.timeout_seconds
-        self.client = cohere.Client(
+        self.client = cohere.ClientV2(
             api_key=cohere_api_key,
             timeout=timeout_seconds,
             client_name=client_name,
             base_url=self.base_url,
         )
-        self.async_client = cohere.AsyncClient(
+        self.async_client = cohere.AsyncClientV2(
             api_key=cohere_api_key,
             client_name=client_name,
             timeout=timeout_seconds,
             base_url=self.base_url,
         )
+        
+        if not self.model:
+            self.model = "command-r-plus"
```

Comment:
let's load default

Reply:
Have made the changes. Also added some logic for mocking the new method in the unit tests